.gitignore

```diff
@@ -17,6 +17,11 @@
 *S_T.inl
 *.bmak
 *.recipe
+*.gpj
+*.int
+*.cmd
```

Comment:
`cmd` is a common extension for windows batch files. There is [one in the tree](https://github.com/DOCGroup/ACE_TAO/blob/master/TAO/orbsvcs/tests/FT_App/replica.cmd), though I don't think it's important. I'm not sure what we can do about this if integrity is using it though.

Reply:
I don't think we need in this PR.  In general, we could remove all tool-specific files from .gitignore and let each user configure their own .git/info/exclude or similar.